patches/net/minecraft/advancements/Advancement.java.patch

```diff
@@ -1,5 +1,13 @@
 --- a/net/minecraft/advancements/Advancement.java
 +++ b/net/minecraft/advancements/Advancement.java
+@@ -54,6 +_,7 @@
+         ),
+         Advancement::validate
+     );
++    public static final Codec<Optional<net.neoforged.neoforge.common.conditions.WithConditions<Advancement>>> CONDITIONAL_ADVANCEMENT_CODEC = net.neoforged.neoforge.common.conditions.ConditionalOps.createConditionalCodecWithConditions(CODEC);
```

Comment:
Is there a reason to keep the ADVANCEMENT in the name of the codec? same question for recipe below

Reply:
Just me not thinking when I moved them out of `NeoForgeExtraCodecs`